platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -160,9 +407,20 @@ int odp_crypto_capability(odp_crypto_capability_t *capa)
 	capa->queue_type_sched = 1;
 
 	capa->ciphers.bit.null       = 1;
-
 	capa->auths.bit.null         = 1;
 
+#ifdef __ARM_FEATURE_AES
+
+#if ODP_DEPRECATED_API
+	capa->ciphers.bit.aes128_gcm = 1;
+	capa->auths.bit.aes128_gcm   = 1;
+#else
+	capa->ciphers.bit.aes_gcm    = 1;
```

Comment:
The aes_gcm capability bits should not depend on ODP_DEPRECATED_API. The current API is always supported (i.e. aes_gcm indicates if AES-GCM is supported). if deprecated APIs are enabled, then AES-GCM support will be visible also in the deprecated capability bits _in addition_ to the current non-deprecated capa bits.

Reply:
I think it wasn't clear to me how ODP_DEPRECATED_API works exactly and this helps to understand. Some other review comments are also around this issue. I will fix it. Thanks Janne!